### PR TITLE
[API Docs] Add three examples for tf.data.Dataset.from_tensor_slices docs

### PR DESCRIPTION
**Pull request by splovyt**
This PR clarifies the use of `tf.data.Dataset.from_tensor_slices` with three concrete examples, which have proven useful in my explanation of the method in one of my projects.

### CODE REVIEW

File: tensorflow/python/data/ops/dataset_ops.py [hunk 1]
```diff
@@ -451,6 +451,28 @@ def from_tensor_slices(tensors):
     # Dictionary structure is also preserved.
     Dataset.from_tensor_slices({"a": [1, 2], "b": [3, 4], "c": [5, 6]})
     # ==> [ {"a": 1, "b": 3, "c": 5}, {"a": 2, "b": 4, "c:" 6} ]
+
+
+    # Example 1: Two tensors can be combined into one Dataset object.
+    features = tf.constant([[1, 3], [2, 3], [2, 1], [1, 2], [3, 3], [3, 2]]) # ==> 6x2 tensor
```
Comment (aaudiber) [inline tensorflow/python/data/ops/dataset_ops.py]: this exceeds the 80 character line limit, can we shorten the example to use a 3x2 tensor?

File: tensorflow/python/data/ops/dataset_ops.py [hunk 2]
```diff
@@ -451,6 +451,28 @@ def from_tensor_slices(tensors):
     # Dictionary structure is also preserved.
     Dataset.from_tensor_slices({"a": [1, 2], "b": [3, 4], "c": [5, 6]})
     # ==> [ {"a": 1, "b": 3, "c": 5}, {"a": 2, "b": 4, "c:" 6} ]
+
+
+    # Example 1: Two tensors can be combined into one Dataset object.
```
Comment (aaudiber) [inline tensorflow/python/data/ops/dataset_ops.py]: maybe remove the "Example 1: " part. The lines above are also examples, so starting the numbering at 1 here looks strange.

**Review by aaudiber — CHANGES_REQUESTED** [review]
Thanks for contributing back these examples @splovyt! I left a couple comments

File: tensorflow/python/data/ops/dataset_ops.py (hunk 1 above)
Comment (splovyt) [reply tensorflow/python/data/ops/dataset_ops.py]: > Thanks for contributing back these examples @splovyt! I left a couple comments

Sure thing, thanks for the review!
I have applied your suggestions; let me know if there's anything else you would like to see changed.

**Comment by splovyt** [comment]
> Thanks for contributing back these examples @splovyt! I left a couple comments

Sure thing, thanks for the review!
I have applied your suggestions; let me know if there's anything else you would like to see changed.

File: tensorflow/python/data/ops/dataset_ops.py [hunk 3]
```diff
@@ -451,6 +451,28 @@ def from_tensor_slices(tensors):
     # Dictionary structure is also preserved.
     Dataset.from_tensor_slices({"a": [1, 2], "b": [3, 4], "c": [5, 6]})
     # ==> [ {"a": 1, "b": 3, "c": 5}, {"a": 2, "b": 4, "c:" 6} ]
+
+
```
Comment (aaudiber) [inline tensorflow/python/data/ops/dataset_ops.py]: please remove the extra newline to make the spacing between examples consistent

**Review by aaudiber — CHANGES_REQUESTED** [review]

File: tensorflow/python/data/ops/dataset_ops.py (hunk 3 above)
Comment (splovyt) [reply tensorflow/python/data/ops/dataset_ops.py]: Fixed, thanks!

**Review by aaudiber — APPROVED** [review]

**Pull request merged**